README.md

```diff
@@ -1095,6 +1130,192 @@ VTP file name. Valid values are a string or the keyword 'default'.
+
+### Type: radius
+
+#### Parameters
+
+##### `name`
+Resource name, not used to manage the device.  Valid value is a string.
+
+##### `enable`
+Enable or disable radius functionality.  Valid values are 'true' or 'false'.
+
+### Type: radius_global
+
+#### Parameters
+
+##### `ensure - Check this one.  Really ensureable?`
+Determines whether or not the config should be present on the device. Valid values are 'present' and 'absent'. Default value is 'present'.
+
+##### `enable`
+**There is an enable property, but it's not implemented in the nxapi provider!**
+
+##### `name`
+Resource identifier, not used to manage the device.  Valid value is a string.
+
+##### `timeout`
+Number of seconds before the timeout period ends.  Valid value is an integer.
+
+##### `retransmit_count`
+Number of times to retransmit.  Valid value is an integer.
+
+##### `key`
+Encryption key (plaintext or in hash form depending on key_format).  Valid value is a string.
+
+##### `key_format`
+Encryption key format [0-7].  Valid value is an integer.
+
+### Type: radius_server
+
+#### Parameters
+
+##### `ensure`
+Determines whether or not the config should be present on the device. Valid values are 'present' and 'absent'. Default value is 'present'.
+
+##### `name`
+Name of the radius server.  Valid value is a string.
+
+##### `auth_port`
+Port number to use for authentication.  Valid value is an integer.
+
+##### `acct_port`
+Port number to use for accounting.  Valid value is an integer.
+
+##### `timeout`
+Number of seconds before the timeout period ends.  Valid value is an integer.
+
+##### `retransmit_count`
+Number of times to retransmit.  Valid value is an integer.
+
+##### `accouting_only`
+Enable this server for accounting only.  Valid values are 'true' or 'false'.
+
+##### `authentication_only`
+Enable this server for authentication only.  Valid values are 'true' or 'false'.
+
+##### `key`
+Encryption key (plaintext or in hash form depending on key_format).  Valid value is a string.
+
+##### `key_format`
+Encryption key format [0-7].  Valid value is an integer.
+
+### Type: syslog_server
+
+#### Parameters
+
+##### `ensure`
+Determines whether or not the config should be present on the device. Valid values are 'present' and 'absent'. Default value is 'present'.
+
+##### `name`
+Hostname or address of the Syslog server.  Valid value is a string.
+
+##### `serverity_level`
+Syslog severity level to log.  Valid value is an integer.
+
+##### `vrf`
+Interface to send syslog data from, e.g. "management".  Valid value is a string.
+
+### Type: syslog_settings
+
+#### Parameters
+
+##### `ensure - Check this one.  Really ensureable?`
+Determines whether or not the config should be present on the device. Valid values are 'present' and 'absent'. Default value is 'present'.
+
+##### `name`
+Hostname or address of the Syslog server.  Valid value is a string.
+
+##### `time_stamp_units`
+The unit to log time values in.  Valid values are 'seconds' and 'milliseconds'.
```

Comment:
`/The unit to log time values in. /The unit of measurement for log time values./`